src/azure-cli/azure/cli/command_modules/sql/_help.py

```diff
@@ -3,612 +3,753 @@
-        - name: Create a database with GeneralPurpose edition, Gen4 hardware, and 1 vcore as a copy of an existing GeneralPurpose database.
-          text: az sql db copy -g mygroup -s myserver -n originalDb --dest-name newDb -f Gen4 -c 1
-    """
+type: command
+short-summary: Create a copy of a database.
+long-summary: A full list of performance level options can be seen by executing `az sql db list-editions -a -o table -l LOCATION`. The copy destination database must have the same edition as the source database, but you can change the edition after the copy has completed.
+examples:
+  - name: Create a database with performance level S0 as a copy of an existing Standard database.
+    text: az sql db copy -g mygroup -s myserver -n originalDb --dest-name newDb --service-objective S0
+  - name: Create a database with GeneralPurpose edition, Gen4 hardware, and 1 vcore as a copy of an existing GeneralPurpose database.
+    text: az sql db copy -g mygroup -s myserver -n originalDb --dest-name newDb -f Gen4 -c 1
+"""
+
 helps['sql db create'] = """
-    type: command
-    short-summary: Create a database.
-    long-summary: A full list of performance level options can be seen by executing `az sql db list-editions -a -o table -l LOCATION`.
-    examples:
-        - name: Create a Standard S0 database.
-          text: az sql db create -g mygroup -s myserver -n mydb --service-objective S0
-        - name: Create a database with GeneralPurpose edition, Gen4 hardware and 1 vcore
-          text: az sql db create -g mygroup -s myserver -n mydb -e GeneralPurpose -f Gen4 -c 1
-        - name: Create a database with zone redundancy enabled
-          text: az sql db create -g mygroup -s myserver -n mydb -z
-        - name: Create a database with zone redundancy explicitly disabled
-          text: az sql db create -g mygroup -s myserver -n mydb -z false
-    """
+type: command
+short-summary: Create a database.
+long-summary: A full list of performance level options can be seen by executing `az sql db list-editions -a -o table -l LOCATION`.
+examples:
+  - name: Create a Standard S0 database.
+    text: az sql db create -g mygroup -s myserver -n mydb --service-objective S0
+  - name: Create a database with GeneralPurpose edition, Gen4 hardware and 1 vcore
+    text: az sql db create -g mygroup -s myserver -n mydb -e GeneralPurpose -f Gen4 -c 1
+  - name: Create a database with zone redundancy enabled
+    text: az sql db create -g mygroup -s myserver -n mydb -z
+  - name: Create a database with zone redundancy explicitly disabled
+    text: az sql db create -g mygroup -s myserver -n mydb -z false
+"""
+
 helps['sql db delete'] = """
-    type: command
-    short-summary: Delete a database.
-    """
+type: command
+short-summary: Delete a database.
+examples:
+  - name: Delete a database. (autogenerated)
+    text: az sql db delete --name MyAzureSQLDatabase --resource-group MyResourceGroup --server myserver
```

Comment:
If MyResourceGroup is standard across all commands, do you want to change mygroup -> MyResourceGroup in the existing examples?